### PR TITLE
Chart changes

### PR DESCRIPTION
**Pull request by Maladie**
**Description**

Changes proposed in this pull request:

- Image bump and Ingress updated for certificates revocation

**Related issue(s)**

Issue #2314 
PR with code changes #2899

### CODE REVIEW

File: resources/application-connector/charts/connector-service/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The Connector Service has the following parameters, that can be set through the
 - **runtimesInfoURL** - URL at which management information for runtimes is available. If not provided, it bases on `connectorServiceHost`.
 - **certificateValidityTime** - Validity time of certificates issued by this service. The default value is 90 days.
 - **central** - Determines whether the Connector Service works in the central mode.
+- **revocationConfigMapName** - Name of the config map containing revoked certificates list.
```
Comment (kazydek) [inline resources/application-connector/charts/connector-service/README.md]:
```suggestion
- **revocationConfigMapName** - Name of the ConfigMap containing revoked certificates list.
```

Comment (akgalwas) [reply resources/application-connector/charts/connector-service/README.md]: Done.

**Pull request merged**